src/Localization/en/translations.json

```diff
@@ -28,6 +28,23 @@
             "SubmissionMessage": "Submitted successfully"
         },
         "SqlX": {
+            "DedicatedGatewayDescription": "Provisioning dedicated gateways for SqlX accounts.",
+            "DedicatedGateway": "Dedicated Gateway",
+            "Enable": "Enable",
+            "Disable": "Disable",
+            "LearnAboutDedicatedGateway": "Learn more about dedicated gateway.",
+            "SKUs": "SKUs",
+            "NumberOfInstances": "Number of instances",
+            "CosmosD4s": "Cosmos.D4s",
```

Comment:
these 4 keys (CosmosD4s, COsmosD8s...) aren't used anywhere. remove them

Reply:
It is being used in ChoiceItem

Reply:
got it